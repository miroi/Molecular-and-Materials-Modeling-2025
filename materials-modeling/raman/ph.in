&INPUTPH
   tr2_ph = 1.0e-12,
   prefix = 'si_scf',
   outdir = './tmp',
   amass(1) = 28.0855,
   epsil = .true.,
   trans = .true.,
   lraman = .true.,
   fildyn = 'si.dyn',
   ldisp = .false.
/
0.0 0.0 0.0
