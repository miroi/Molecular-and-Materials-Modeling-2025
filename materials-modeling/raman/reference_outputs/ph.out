
     Program PHONON v.7.4.1 starts on 15Jul2025 at 12:22: 6 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Serial version
     24066 MiB available memory on the printing compute node when the environment starts

     Waiting for input...
     Reading input from standard input
      Title line not specified: using 'default'.

     Reading xml data from directory:

     ./tmp/si_scf.save/


     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PZ
                           (   1   1   0   0   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         859     433    127                16889     5985     965

     Using Slab Decomposition

     Reading collected, re-writing distributed wavefunctions in ./tmp/

     Number and degeneracy of irreps per q-point
       N         xq(1)         xq(2)         xq(3)   N irreps
       1   0.000000000   0.000000000   0.000000000       2
        3   3

     Calculation of q =    0.0000000   0.0000000   0.0000000

     default                                                                    

     bravais-lattice index     =            0
     lattice parameter (alat)  =       7.2558  a.u.
     unit-cell volume          =     270.1072 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     kinetic-energy cut-off    =      30.0000  Ry
     charge density cut-off    =     240.0000  Ry
     convergence threshold     =      1.0E-12
     beta                      =       0.7000
     number of iterations used =            4
     Exchange-correlation= PZ
                           (   1   1   0   0   0   0   0)


     celldm(1)=    7.25577  celldm(2)=    0.00000  celldm(3)=    0.00000
     celldm(4)=    0.00000  celldm(5)=    0.00000  celldm(6)=    0.00000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  0.0000  0.7071  0.7071 )  
               a(2) = (  0.7071  0.0000  0.7071 )  
               a(3) = (  0.7071  0.7071  0.0000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -0.7071  0.7071  0.7071 )  
               b(2) = (  0.7071 -0.7071  0.7071 )  
               b(3) = (  0.7071  0.7071 -0.7071 )  


     Atoms inside the unit cell: 

     Cartesian axes

     site n.  atom      mass           positions (alat units)
        1  Si     28.0855   tau(    1) = (    0.00000    0.00000    0.00000  )
        2  Si     28.0855   tau(    2) = (    0.35355    0.35355    0.35355  )

     Computing dynamical matrix for 
                    q = (   0.0000000   0.0000000   0.0000000 )

     49 Sym.Ops. (with q -> -q+G )


     G cutoff =  320.0508  (  16889 G-vectors)     FFT grid: ( 36, 36, 36)
     G cutoff =  160.0254  (   5985 G-vectors)  smooth grid: ( 32, 32, 32)
     number of k points=     3
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.2500000
        k(    2) = (  -0.3535534  -0.3535534   0.3535534), wk =   1.0000000
        k(    3) = (  -0.7071068   0.0000000   0.0000000), wk =   0.7500000

     PseudoPot. # 1 for Si read from file:
     ./Si.pz-vbc.UPF
     MD5 check sum: 6dfa03ddd5817404712e03e4d12deb78
     Pseudo is Norm-conserving, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  431 points,  2 beta functions with: 
                l(1) =   0
                l(2) =   1

     Mode symmetry, O_h (m-3m)  point group:


     Electric field:
     Dielectric constant
     Born effective charges as d Force / d E
     Raman tensor


     Atomic displacements:
     There are    2 irreducible representations

     Representation     1      3 modes -  To be done

     Representation     2      3 modes -  To be done



     Alpha used in Ewald sum =   2.5000
     PHONON       :      0.04s CPU      0.04s WALL


     Electric Fields Calculation

      iter #   1 total cpu time :     0.5 secs   av.it.:   6.7
      thresh= 1.000E-02 alpha_mix =  0.700 |ddv_scf|^2 =  5.375E-07

      iter #   2 total cpu time :     0.8 secs   av.it.:  10.0
      thresh= 7.331E-05 alpha_mix =  0.700 |ddv_scf|^2 =  4.041E-08

      iter #   3 total cpu time :     1.1 secs   av.it.:  10.3
      thresh= 2.010E-05 alpha_mix =  0.700 |ddv_scf|^2 =  1.765E-09

      iter #   4 total cpu time :     1.4 secs   av.it.:  10.1
      thresh= 4.201E-06 alpha_mix =  0.700 |ddv_scf|^2 =  1.251E-11

      iter #   5 total cpu time :     1.6 secs   av.it.:  10.3
      thresh= 3.537E-07 alpha_mix =  0.700 |ddv_scf|^2 =  1.109E-15

     End of electric fields calculation

          Dielectric constant in cartesian axis 

          (      59.855528368       0.000000000      -0.000000000 )
          (      -0.000000000      59.855528368       0.000000000 )
          (      -0.000000000       0.000000000      59.855528368 )

          Effective charges (d Force / dE) in cartesian axis without acoustic sum rule applied (asr)

           atom    1  Si    Mean Z*:       -5.60370
      Ex  (       -5.60370       -0.00000       -0.00000 )
      Ey  (       -0.00000       -5.60370       -0.00000 )
      Ez  (       -0.00000        0.00000       -5.60370 )
           atom    2  Si    Mean Z*:       -5.60370
      Ex  (       -5.60370       -0.00000       -0.00000 )
      Ey  (       -0.00000       -5.60370       -0.00000 )
      Ez  (       -0.00000       -0.00000       -5.60370 )

          Effective charges Sum: Mean:      -11.20739
            -11.20739       -0.00000       -0.00000
             -0.00000      -11.20739       -0.00000
             -0.00000       -0.00000      -11.20739

          Effective charges (d Force / dE) in cartesian axis with asr applied: 
           atom    1  Si    Mean Z*:       -0.00000
      E*x (       -0.00000        0.00000        0.00000 )
      E*y (        0.00000       -0.00000        0.00000 )
      E*z (       -0.00000        0.00000       -0.00000 )
           atom    2  Si    Mean Z*:        0.00000
      E*x (        0.00000       -0.00000       -0.00000 )
      E*y (        0.00000        0.00000       -0.00000 )
      E*z (        0.00000       -0.00000        0.00000 )

     Calling punch_plot_e
     Writing on file  drho                                                                                                                                                                                                                                                            

     Computing Pc [DH,Drho] |psi> 

     Derivative coefficient:  0.001000    Threshold: 1.00E-12
     Non-scf  u_k: avg # of iterations = 16.9
     Non-scf Du_k: avg # of iterations = 24.7

          Dielectric constant from finite-differences

          (      59.847283849       0.000000000      -0.000000000 )
          (       0.000000000      59.847283849       0.000000000 )
          (      -0.000000000       0.000000000      59.847283849 )

     Computing Second order response 


      iter #   1   av.it.:   9.1
      thresh= 1.000E-02 alpha_mix =  0.700 |ddv_scf|^2 =  1.109E-04


      iter #   2   av.it.:   9.3
      thresh= 1.053E-03 alpha_mix =  0.700 |ddv_scf|^2 =  6.660E-07


      iter #   3   av.it.:   9.9
      thresh= 8.161E-05 alpha_mix =  0.700 |ddv_scf|^2 =  1.491E-08


      iter #   4   av.it.:   9.3
      thresh= 1.221E-05 alpha_mix =  0.700 |ddv_scf|^2 =  4.298E-10


      iter #   5   av.it.:   9.4
      thresh= 2.073E-06 alpha_mix =  0.700 |ddv_scf|^2 =  7.835E-12


      iter #   6   av.it.:   9.3
      thresh= 2.799E-07 alpha_mix =  0.700 |ddv_scf|^2 =  7.961E-15

          Raman tensor (au^-1) in cartesian axis 

           atom      1
          (       0.000000000      -0.000000000      -0.000000000 )
          (      -0.000000000       0.000000000     -25.222643770 )
          (      -0.000000000     -25.222643770      -0.000000000 )

          (      -0.000000000       0.000000000     -25.222643770 )
          (       0.000000000      -0.000000000       0.000000000 )
          (     -25.222643770       0.000000000       0.000000000 )

          (       0.000000000     -25.222643770       0.000000000 )
          (     -25.222643770       0.000000000      -0.000000000 )
          (       0.000000000      -0.000000000      -0.000000000 )

           atom      2
          (      -0.000000000       0.000000000      -0.000000000 )
          (      -0.000000000       0.000000000      25.222643770 )
          (       0.000000000      25.222643770      -0.000000000 )

          (      -0.000000000       0.000000000      25.222643770 )
          (       0.000000000      -0.000000000      -0.000000000 )
          (      25.222643770       0.000000000       0.000000000 )

          (      -0.000000000      25.222643770       0.000000000 )
          (      25.222643770      -0.000000000      -0.000000000 )
          (      -0.000000000       0.000000000       0.000000000 )


     Raman tensor (A^2)

     atom #    1    pol.  1
      0.312884427362E-13     -0.328635222311E-13     -0.990445441374E-14
     -0.328635222311E-13      0.312884427362E-13     -0.151816546055E+03
     -0.990445441374E-14     -0.151816546055E+03     -0.542475105536E-13
     atom #    1    pol.  2
     -0.328635222311E-13      0.312884427362E-13     -0.151816546055E+03
      0.259424456556E-13     -0.275175251505E-13      0.613353682809E-14
     -0.151816546055E+03      0.114795339087E-13      0.740564193811E-13
     atom #    1    pol.  3
      0.328635222311E-13     -0.151816546055E+03      0.312884427362E-13
     -0.151816546055E+03      0.542475105536E-13     -0.328635222311E-13
      0.990445441374E-14     -0.328635222311E-13     -0.526724310586E-13
     atom #    2    pol.  1
     -0.154867134186E-13      0.589727490385E-14     -0.272812632263E-13
     -0.479471925737E-14      0.165892690651E-13      0.151816546055E+03
      0.261787075798E-13      0.151816546055E+03     -0.261787075798E-13
     atom #    2    pol.  2
     -0.208327104992E-13      0.326272603069E-13      0.151816546055E+03
      0.112432719845E-13     -0.208327104992E-13     -0.551277823236E-15
      0.151816546055E+03      0.368707017410E-13      0.589727490385E-14
     atom #    2    pol.  3
     -0.326272603069E-13      0.151816546055E+03      0.540112486293E-13
      0.151816546055E+03     -0.112432719845E-13     -0.315247046604E-13
     -0.101407163380E-13      0.112432719845E-13      0.101407163380E-13


     Representation #   1 modes #   1   2   3

     Self-consistent Calculation

      iter #   1 total cpu time :     9.7 secs   av.it.:   4.6
      thresh= 1.000E-02 alpha_mix =  0.700 |ddv_scf|^2 =  2.688E-08

      iter #   2 total cpu time :    10.0 secs   av.it.:   8.4
      thresh= 1.640E-05 alpha_mix =  0.700 |ddv_scf|^2 =  2.310E-10

      iter #   3 total cpu time :    10.2 secs   av.it.:   8.9
      thresh= 1.520E-06 alpha_mix =  0.700 |ddv_scf|^2 =  6.938E-12

      iter #   4 total cpu time :    10.5 secs   av.it.:   8.9
      thresh= 2.634E-07 alpha_mix =  0.700 |ddv_scf|^2 =  8.461E-15

     End of self-consistent calculation

     Convergence has been achieved 


     Representation #   2 modes #   4   5   6

     Self-consistent Calculation

      iter #   1 total cpu time :    10.7 secs   av.it.:   5.2
      thresh= 1.000E-02 alpha_mix =  0.700 |ddv_scf|^2 =  3.072E-08

      iter #   2 total cpu time :    11.0 secs   av.it.:  10.2
      thresh= 1.753E-05 alpha_mix =  0.700 |ddv_scf|^2 =  2.860E-09

      iter #   3 total cpu time :    11.3 secs   av.it.:  10.3
      thresh= 5.348E-06 alpha_mix =  0.700 |ddv_scf|^2 =  1.302E-11

      iter #   4 total cpu time :    11.6 secs   av.it.:  10.6
      thresh= 3.608E-07 alpha_mix =  0.700 |ddv_scf|^2 =  3.566E-14

     End of self-consistent calculation

     Convergence has been achieved 

     Number of q in the star =    1
     List of q in the star:
          1   0.000000000   0.000000000   0.000000000

          Dielectric constant in cartesian axis 

          (      59.847283849       0.000000000      -0.000000000 )
          (       0.000000000      59.847283849       0.000000000 )
          (      -0.000000000       0.000000000      59.847283849 )

          Effective charges (d Force / dE) in cartesian axis without acoustic sum rule applied (asr)

           atom    1  Si    Mean Z*:       -5.60370
      Ex  (       -5.60370       -0.00000       -0.00000 )
      Ey  (       -0.00000       -5.60370       -0.00000 )
      Ez  (       -0.00000        0.00000       -5.60370 )
           atom    2  Si    Mean Z*:       -5.60370
      Ex  (       -5.60370       -0.00000       -0.00000 )
      Ey  (       -0.00000       -5.60370       -0.00000 )
      Ez  (       -0.00000       -0.00000       -5.60370 )

          Effective charges Sum: Mean:      -11.20739
            -11.20739       -0.00000       -0.00000
             -0.00000      -11.20739       -0.00000
             -0.00000       -0.00000      -11.20739

          Effective charges (d Force / dE) in cartesian axis with asr applied: 
           atom    1  Si    Mean Z*:       -0.00000
      E*x (       -0.00000        0.00000        0.00000 )
      E*y (        0.00000       -0.00000        0.00000 )
      E*z (       -0.00000        0.00000       -0.00000 )
           atom    2  Si    Mean Z*:        0.00000
      E*x (        0.00000       -0.00000       -0.00000 )
      E*y (        0.00000        0.00000       -0.00000 )
      E*z (        0.00000       -0.00000        0.00000 )

     Diagonalizing the dynamical matrix

     q = (    0.000000000   0.000000000   0.000000000 ) 

 **************************************************************************
     freq (    1) =       0.523480 [THz] =      17.461428 [cm-1]
     freq (    2) =       0.523480 [THz] =      17.461428 [cm-1]
     freq (    3) =       0.523480 [THz] =      17.461428 [cm-1]
     freq (    4) =      17.213659 [THz] =     574.185872 [cm-1]
     freq (    5) =      17.213659 [THz] =     574.185872 [cm-1]
     freq (    6) =      17.213659 [THz] =     574.185872 [cm-1]
 **************************************************************************

     Mode symmetry, O_h (m-3m)  point group:

     freq (   1-   3) =         17.5  [cm-1]   --> T_1u G_15  G_4- I  
     freq (   4-   6) =        574.2  [cm-1]   --> T_2g G_25' G_5+ R  

     PHONON       :     11.47s CPU     11.63s WALL

     INITIALIZATION: 
     phq_setup    :      0.00s CPU      0.00s WALL (       1 calls)
     phq_init     :      0.01s CPU      0.01s WALL (       1 calls)

     phq_init     :      0.01s CPU      0.01s WALL (       1 calls)
     init_vloc    :      0.01s CPU      0.01s WALL (       1 calls)
     init_us_1    :      0.00s CPU      0.00s WALL (       1 calls)
     init_us_2    :      0.01s CPU      0.01s WALL (     108 calls)

     DIELECTRIC CONSTANT AND EFFECTIVE CHARGES:
     solve_e      :      1.57s CPU      1.59s WALL (       1 calls)
     dielec       :      0.00s CPU      0.00s WALL (       1 calls)
     zstar_eu     :      0.03s CPU      0.04s WALL (       1 calls)

     RAMAN COEFFICIENTS, THIRD-ORDER CHI:
     dhdrhopsi    :      4.34s CPU      4.38s WALL (       1 calls)
     dvpsi_e2     :      0.22s CPU      0.22s WALL (       1 calls)
     solve_e2     :      3.01s CPU      3.06s WALL (       1 calls)

     DYNAMICAL MATRIX:
     dynmat0      :      0.01s CPU      0.01s WALL (       1 calls)
     dynmat_us    :      0.01s CPU      0.01s WALL (       1 calls)
     d2ionq       :      0.00s CPU      0.00s WALL (       1 calls)
     phqscf       :      2.08s CPU      2.12s WALL (       1 calls)
     dynmatrix    :      0.00s CPU      0.00s WALL (       1 calls)

     phqscf       :      2.08s CPU      2.12s WALL (       1 calls)
     solve_linter :      2.07s CPU      2.11s WALL (       2 calls)
     sth_kernel   :      2.00s CPU      2.00s WALL (      13 calls)
     h_prec       :      0.00s CPU      0.00s WALL (     264 calls)
     apply_dpot_b :      0.13s CPU      0.13s WALL (      90 calls)
     ortho        :      0.00s CPU      0.00s WALL (     180 calls)
     cgsolve      :      7.30s CPU      7.35s WALL (     342 calls)
     incdrhoscf   :      0.33s CPU      0.33s WALL (     225 calls)
                                        0.00s GPU  (     225 calls)
     dv_of_drho   :      0.12s CPU      0.13s WALL (      84 calls)
     mix_pot      :      0.08s CPU      0.13s WALL (      19 calls)
     drhodv       :      0.01s CPU      0.01s WALL (       2 calls)
     symdvscf     :      1.18s CPU      1.18s WALL (      13 calls)
     com_dvloc    :      0.04s CPU      0.04s WALL (     120 calls)
                                        0.00s GPU  (     120 calls)
     addcore      :      0.00s CPU      0.00s WALL (      24 calls)
                                        0.00s GPU  (      24 calls)

     dvqpsi_us    :      0.19s CPU      0.21s WALL (     108 calls)
                                        0.00s GPU  (     108 calls)
     dvqpsi_us_on :      0.01s CPU      0.01s WALL (     108 calls)

     add_vuspsi   :      0.10s CPU      0.10s WALL (    6230 calls)


      General routines
     calbec       :      0.13s CPU      0.13s WALL (    7763 calls)
     fft          :      0.16s CPU      0.17s WALL (     422 calls)
     ffts         :      0.08s CPU      0.09s WALL (     286 calls)
     fftw         :      7.12s CPU      7.17s WALL (   44946 calls)
     davcio       :      0.07s CPU      0.10s WALL (    1260 calls)
     write_rec    :      0.02s CPU      0.03s WALL (      21 calls)

      Additional routines
     ch_psi       :      7.17s CPU      7.22s WALL (    5080 calls)
     last         :      0.14s CPU      0.14s WALL (    5080 calls)
     Hesh         :      0.01s CPU      0.01s WALL (    5080 calls)
     ch_psi_all_k :      0.12s CPU      0.12s WALL (    5080 calls)
                                        0.00s GPU  (    5080 calls)
     h_psi_bgrp   :      7.45s CPU      7.50s WALL (    6230 calls)
     h_psi        :      7.45s CPU      7.50s WALL (    6230 calls)
     h_psi:pot    :      7.44s CPU      7.48s WALL (    6230 calls)
     h_psi:calbec :      0.12s CPU      0.12s WALL (    6230 calls)
     s_psi_bgrp   :      0.01s CPU      0.01s WALL (   12608 calls)


     PHONON       :     11.47s CPU     11.63s WALL


   This run was terminated on:  12:22:18  15Jul2025            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
